packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -59,5 +55,13 @@ export const useModalAttributes = (
     deloser: {},
   });
 
+  React.useEffect(() => {
```

Comment:
AFAIK tabster has dummy inputs, are they used there? If so `useLayoutEffect` should be potentially safer

Reply:
Resolving for now, there are other bugs that need to be fixed and v9 currently does not support React 18 anyway. Tracking tabster support for 18 in #24085